cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-common/management-credential-step/ManagementCredentials.tsx

```diff
@@ -153,6 +170,57 @@ function ManagementCredentials(props: Partial<StepProps>) {
         }
     };
 
+    useEffect(() => {
+        retrieveOsImages(awsState[STORE_SECTION_FORM].REGION);
```

Comment:
Before calling `retrieveOsImages`, perhaps we should check if `awsState[STORE_SECTION_FORM].REGION` has a value?

Reply:
Yes. will do